core/tests/protractor_utils/creatorDashboard.js

```diff
@@ -13,20 +13,28 @@
 // limitations under the License.
 
 /**
- * @fileoverview Utilities for interacting with the creator dashboard page,
- * for use in Protractor tests.
+ * @fileoverview Page object for the creator dashboard, for use in Protractor
+ * tests.
  */
 
-var getNumberOfFeedbackMessages = function() {
-  return element(by.css('.protractor-test-exploration-feedback-count')).
-    getText().then(function(text) {
+var CreatorDashboard = function() {
+  var CREATOR_DASHBOARD_URL = '/creator_dashboard';
+  var feedbackCount =
```

Comment:
explorationFeedbackCount (there is separate stuff on that page for collections)

Reply:
Done.